js/qmwp-add-measurement.js

```diff
@@ -583,14 +583,18 @@ jQuery(document).ready(function () {
                     variableValue = variable.mostCommonValue;
                 }
 
+                setInputValues(variableUnit, variableValue);
+
             }
 
-            if (variableUnit === null) {
-                return;
+
+            function setInputValues(unit, value) {
```

Comment:
It used few lines before: https://github.com/Abolitionist-Project/QuantiModo-WordPress-Plugin/pull/170#discussion-diff-45116686R586